libs/s25main/desktops/dskCredits.cpp

```diff
@@ -454,11 +465,10 @@ void dskCredits::DrawBobs()
                     bob.direction = libsiedler2::ImgDir::E;
             }
         }
+    } else if(tmrId == ID_tmrPage)
+    {
+        GotoNextPage();
```

Comment:
so we can't go back on right click anymore? why?

Reply:
We can, nothing changed there

Reply:
ah forget it

Reply:
ah right, we have different styles everywhere :-D